README.md

```diff
@@ -2,7 +2,7 @@
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 [![Discord](https://img.shields.io/discord/699608417039286293?style=flat-square)](https://discord.gg/jZQs6Wu)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
-Raise an issue to join the EddieHub GitHub community.
+Raise an [issue to join the EddieHub GitHub community](https://github.com/EddieHubCommunity/support/issues/new?assignees=&labels=invite+me+to+the+organisation&template=invitation.md&title=Please+invite+me+to+the+GitHub+Community+Organization).
```

Comment:
Which is better, `issue to join the EddieHub GitHub community` or `issue to join the EddieHub GitHub organisation`?

Reply:
> Which is better, `issue to join the EddieHub GitHub community` or `issue to join the EddieHub GitHub organisation`?

Think organization is the more accurate term 🤔

Reply:
According to me "Community " sounds more helpful and friendly